geonode/maps/urls.py

```diff
@@ -49,6 +49,7 @@
     url(r'^new/data$', 'new_map_json', name='new_map_json'),
     url(r'^checkurl/?$', 'ajax_url_lookup'),
     url(r'^snapshot/create/?$', 'snapshot_create'),
+    url(r'^new/copy/(?P<mapid>[^/]+)$', existing_map_view, name='copy_map'),
```

Comment:
It is ok to define this, but if it's going to be used by geoserver too, then you need to make sure it works.